src/Core/Validation/FieldValidation/FieldValidatorsTrait.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace SilverStripe\Core\Validation\FieldValidation;
+
+use RuntimeException;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Validation\FieldValidation\FieldValidationInterface;
+use SilverStripe\Core\Validation\ValidationResult;
+use SilverStripe\Forms\FormField;
+
+trait FieldValidatorsTrait
+{
+    /**
+     * FieldValidators configuration for the field, which is either a FormField or DBField
+     *
+     * Each item in the array can be one of the following
+     * a) MyFieldValidator::class,
+     * b) MyFieldValidator::class => [null, 'getMyArg'],
+     * c) MyFieldValidator::class => null,
+     *
+     * a) Will create a FieldValidator and pass the name and value of the field as args to the constructor
+     * b) Will create a FieldValidator and pass the name, value, make a pass additional args, calling each
```

Comment:
I've updated the comment, particularly for b) which was a mess

> Would you be upset

Yes :-)

Reply:
I think it's a very big improvement in terms of making the API intuitive. That `null` is super confusing. I don't understand why you'd be upset about it but at this stage just leave as is simply because I can't see you agreeing with me here.